user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md

```diff
@@ -4,11 +4,16 @@ uid: Controlling_remote_access
 
 # Controlling remote access with the Admin app
 
-When a DMS is cloud-connected, you can use the URL created for you during registration (e.g. ``https://ziine-skyline.on.dataminer.services/``) to access the DataMiner web apps from anywhere.
+When a DMS is cloud-connected, you can use the URL created for you during registration (e.g. ``https://ziine-skyline.on.dataminer.services/``) to access the DataMiner web apps from anywhere. See for example [Accessing the Monitoring app via the DataMiner Cloud Platform](xref:Accessing_the_Monitoring_app#accessing-the-monitoring-app-via-the-dataminer-cloud-platform).
 
 ## Finding the remote access URL
 
-After you have registered, you can find the URL on the dataminer.services home page. In the DataMiner Systems section, the URL is displayed right underneath the name of the connected DataMiner System. You can also directly navigate to the URL from this section.
+After you have registered, you can find the URL on the *Overview* page for the DataMiner System in the Cloud Admin app. The URL is displayed right underneath the name of the connected DataMiner System. You can also directly navigate to the URL from this section.
```

Comment:
To be consistent , I think we have used DCP Admin before (and in the UI on https://dataminer.services) 
```suggestion
After you have registered, you can find the URL on the *Overview* page for the DataMiner System in the DCP Admin app. The URL is displayed right underneath the name of the connected DataMiner System. You can also directly navigate to the URL from this section.
```